model-optimizer/extensions/ops/ctc_greedy_decoder.py

```diff
@@ -40,11 +41,25 @@ def supported_attrs(self):
         ]
 
     @staticmethod
-    def ctc_greedy_decoder_infer(node: Node):
-        outn = node.out_node(0)
-        inn = node.in_node(0)
-        inn2 = node.in_node(1)
-        outn.shape = np.ones(4, dtype=np.int)
-        assert inn.shape[1] == inn2.shape[1], 'Batch for CTCGreedyDecoder should be the same in both inputs'
-        outn.shape[0] = inn.shape[1]
-        outn.shape[1] = inn.shape[0]
+    def infer(node: Node):
+        node_name = node.soft_get('name', node.id)
+        assert len(node.in_nodes()) == 2, \
```

Comment:
check for connected input ports instead of input data nodes please

Reply:
Fixed.